packages/liveblocks-react/src/ai.tsx

```diff
@@ -35,30 +39,24 @@ export const RegisterAiKnowledge = memo(function RegisterAiKnowledge(
 ) {
   const layerId = useId();
   const ai = useAi();
-  const { description, value } = props;
-
-  const [layerKey, setLayerKey] = useState<
-    ReturnType<typeof ai.registerKnowledgeLayer> | undefined
-  >();
```

Comment:
I cannot remember why this extra `layerKey` state var was needed. Maybe it had to do with SSR or StrictMode? I'd have to take a look at it again to refresh my memory. Have you tried it out in real examples with this change yet?